packages/utils/src/schemas/DigestArticleFrontmatterSchema.ts

```diff
@@ -12,6 +12,7 @@ export const DigestArticleFrontmatterSchema = DynamicBaseDataSchema.extend({
       'last-name': z.string(),
       image: ImagePropsSchema.optional(),
       company: z.string(),
+      bio: z.string().optional(),
```

Comment:
Isn't optional for FFDW - We might have to extend this Schema.

Reply:
Ah right, so I guess we have to move it from the shared library, like with the Blog Posts

Reply:
No, I don't think we need to move it. FF doesn't use `bio` so we could just extend this Schema with `bio` for FFDW?

Reply:
Actually, I'm really sorry. Come to think of it, I think we should make `bio` optional .. 😔